app/dashboard/src/components/AriaComponents/Text/Text.tsx

```diff
@@ -48,8 +48,12 @@ export const TEXT_STYLE = twv.tv({
     variant: {
       custom: '',
       body: 'text-xs leading-[20px] before:h-[1px] after:h-[3px] font-medium',
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      'body-sm': 'text-[10.5px] leading-[16px] before:h-[0.5px] after:h-[2.5px] font-medium',
```

Comment:
i guess personally i would prefer adding new text size(s) for consistency. same applies to the new smaller font size below

see: <https://github.com/enso-org/enso/pull/10924/files?w=1#diff-e2514dc865111604d2abd920738e7f4f3827387245dae778d88e378590288c65R68>